pkg/Makefile

```diff
@@ -0,0 +1,88 @@
+# Variables required for this Makefile
+PKG_DIR = $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
+TOP_DIR = $(PKG_DIR)/../bin
+BUILD_DIR = $(PKG_DIR)/build
+TARGET_DIR = $(PKG_DIR)/target
+CONFIG_DIR = /etc/aerospike
+# Package variables
+BUILD_DISTRO ?= $(shell .github/packaging/common/os_version.sh)
+NAME = "aerospike-$(PACKAGE_NAME)"
+MAINTAINER = "Aerospike"
+VERSION = $(shell git describe --tags --always)
+DESCRIPTION = "ASBackup, a tool for backing up and restoring data for Aerospike database."
+LICENSE = "Apache License 2.0"
+URL = "https://github.com/aerospike/$(PACKAGE_NAME)"
+VENDOR = "Aerospike, Inc."
+ARCH = $(shell uname -m)
```

Comment:
You probably want `:=` for many of these - otherwise it will execute the shell commands every time the variable is used.